lib/sem/credentials.rb

```
@@ -0,0 +1,13 @@
+module Sem
+  class Credentials
+    PATH = File.expand_path("~/.sem/credentials").freeze
+
+    def self.write(auth_token)
+      File.write(PATH, auth_token)
+    end
```

Please raise the appropriate error if the file is not available.